tutorials/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Earth Engine for Public Health Land Surface Temperature Example Uganda
+description: This tutorial shows how to describe, visualize, and export Earth Engine exposure data for public health research.
+author: hiyer20
+tags: comma-separated, lowercase, list, of, related, keywords
+date_published: 2019-09-17
+---
+
+
+---
+title: Earth Engine for Public Health Land Surface Temperature in Uganda Example
+
+---
```

Comment:
Each tutorial should only have one front matter block; you can remove this duplicate one.